.gitignore

```diff
@@ -0,0 +1,5 @@
+local
+
+temp.txt
+
+.DS_store
```

Comment:
you've added DS_store in gitignore after commiting it. So it is in your PR anyway. 

In order to ignore files you need to add them into gitignore before you've managed to commit them. 
Try to fix this issue